crates/cargo-platform/src/cfg.rs

```diff
@@ -235,6 +250,25 @@ impl<'a> Parser<'a> {
                 self.eat(&Token::RightParen)?;
                 Ok(CfgExpr::Not(Box::new(e)))
             }
+            Some(Ok(Token::Ident(false, "version"))) => {
+                self.t.next();
+                self.eat(&Token::LeftParen)?;
```

Comment:
Can you have `--cfg version=hello` and so need to support `#[cfg(version)1.30.0)]` and `#[cfg(version = "hello")]`?

Reply:
Looking at the updated RFC, I assume the answer is no?

Reply:
I thought the stabilization report said that `--cfg version=hello` would still work

> #[cfg(version = "1.20.0")] is not supported, and there is a warning of the unexpected_cfgs lint (but no compilation error)

(not supported within the context of this feature, `unexpected_cfgs` firing because `--check-cfg` wasn't specified for `version`)

> #[cfg(version = "1.20.0")] acts as if the cfg was false due to the wrong syntax, even if the compiler version is above the specified version

(again, framed within this feature; it could be true if `--cfg version="1.20.0"` was specified)

Reply:
I pushed a change so that `version = …` is ignored by the resolver now. I also raise a warning on it.

In what context could `--cfg version="1.23.4"` be used? I never specified a `cfg` flag. Where is that parsed? How should I handle it?

Reply:
> In what context could --cfg version="1.23.4" be used? I never specified a cfg flag. Where is that parsed? How should I handle it?

More so the point is that this change should be agnostic to `--cfg version="1.23.4"` and should not change the behavior of `cfg(version = "1.23.4")`.